api/lib/infrastructure/data/knowledge-element.js

```diff
@@ -8,7 +8,7 @@ module.exports = Bookshelf.model('KnowledgeElement', {
   tableName: 'knowledge-elements',
   hasTimestamps: ['createdAt', null],
 
-  domainModelName: 'SmartPlacementKnowledgeElement',
+  domainModelName: 'KnowledgeElement',
```

Comment:
Je crois que cette ligne est obsolète du coup. Elle visait à faire le lien entre le nom du modèle Bookshelf et le nom du modèle Domain quand ceux ci sont différents pour le `bookshelf-to-domain-converter`

Reply:
La ligne oui mais par contre, ne surtout pas supprimer le code du `bookshelf-to-domain converter` qui gère cette feature parce que dans l'absolu, ils ont le droit d'être différent !

Reply:
Ok, bon à savoir, je supprime cette ligne